WordOfTheDay.Repository/WordsRepository.cs

```diff
@@ -22,13 +22,19 @@ private static DateTime DateToday
         }
         public async Task<WordCount> WordOfTheDay()
         {
+
             var wordOfTheDay = await _context.Words
                 .LaterThan(DateToday)
-                .FindWordOfTheDay();
-            
+                .GroupBy(word => word.Text, (text, words) => new WordCount { Word = text, Count = words.Count(word => word.Text == text) })
```

Comment:
Can you move this logic to extension method? And use it also for calculating close words?

Reply:
There is some issue described in the Skype

Reply:
done